cj-nmc-daemon/src/main/java/org/consensusj/namecoin/daemon/config/NamecoinConfig.java

```diff
@@ -5,20 +5,27 @@
 import com.msgilligan.bitcoinj.json.conversion.RpcServerModule;
 import com.msgilligan.bitcoinj.rpcserver.BitcoinJsonRpc;
 import com.msgilligan.bitcoinj.spring.service.PeerGroupService;
+import com.msgilligan.bitcoinj.spring.service.WalletAppKitService;
+import org.bitcoinj.core.Context;
 import org.bitcoinj.core.NetworkParameters;
 import org.bitcoinj.net.discovery.DnsDiscovery;
 import org.bitcoinj.net.discovery.PeerDiscovery;
+import org.bitcoinj.kits.WalletAppKit;
 import org.libdohj.params.NamecoinMainNetParams;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import java.io.File;
 import java.io.FileNotFoundException;
 
 /**
  * Spring configuration for namecoinj, Namecoin services, and JSON-RPC server
  */
 @Configuration
 public class NamecoinConfig {
+    private Context context;
+    private WalletAppKit kit;
```

Comment:
The same simplifying changes (relying on Spring to create singletons) requested in `BitcoinConfig` should be made in this class.